functions.sh

```diff
@@ -565,6 +565,22 @@ dolphin_init() {
   dir_prep "$rdhome/saves/wii/dolphin" "/var/data/dolphin-emu/Wii"
 }
 
+primehack_init() {
+  echo "----------------------"
+  echo "Initializing Primehack"
+  echo "----------------------"
+  mkdir -pv /var/config/primehack/
+  cp -fvr "$emuconfigs/primehack/"* /var/config/primehack/
```

Comment:
There doesn't appear to be a `primehack` folder in the repo under `$emuconfigs/primehack/`, is this intended or am I missing something?